ganttproject/src/net/sourceforge/ganttproject/GPTreeTableBase.java

```diff
@@ -255,9 +255,12 @@ public int compare(ColumnImpl left, ColumnImpl right) {
           if (test1 != 0) {
             return test1;
           }
-          if (!left.getStub().isVisible() && !right.getStub().isVisible()) {
+          if (!left.getStub().isVisible() && !right.getStub().isVisible() && left.getName()!=null && right.getName()!=null) {
```

Comment:
does it fix any issues? Looks unrelated to colors.
Pls surround comparison operators with spaces: `getName() != null`